.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,19 +2,24 @@
 <!-- If your pull request is not ready for review yet, create a draft pull request! -->
 
 ### Checklist
-- [ ] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
-- [ ] Server: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
-- [ ] Server: I added multiple integration tests (Spring) related to the features (with a high test coverage)
-- [ ] Server: I added `@PreAuthorize` and check the course groups for all new REST Calls (security)
-- [ ] Server: I implemented the changes with a good performance and prevented too many database calls
-- [ ] Server: I documented the Java code using JavaDoc style.
-- [ ] Client: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
-- [ ] Client: I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-testing/).
-- [ ] Client: I added `authorities` to all new routes and check the course groups for displaying navigation elements (links, buttons)
-- [ ] Client: I documented the TypeScript code using JSDoc style.
-- [ ] Client: I added multiple screenshots/screencasts of my UI changes
-- [ ] Client: I translated all newly inserted strings into English and German.
+#### General
+<!-- If testing it locally is acceptable, you may remove the first checkmark. If you are unsure, please test on the test servers. -->
+- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
+- [ ] I tested **all** changes and their related features with **all** corresponding user types locally. I used a local CI setup if necessary.
```

Comment:
What exactly is meant with local CI setup? Do you refer to a programming exercise environment based on Bamboo, Bitbucket and Jira? 
Maybe we should have two new options:
- [ ] I tested **all** changes and their related features with **all** corresponding user types locally. I used a local programming exercise environment based on Bamboo, Bitbucket and Jira.
- [ ] I tested **all** changes and their related features with **all** corresponding user types locally. I used a local programming exercise environment based on Jenkins and Gitlab.